js/letters.js

```diff
@@ -16,8 +16,22 @@ var counter = {
 }
 
 function countLetters(counter, sample_text){
-  // FIX ME
-}
+  
+  //base case
+  if (sample_text.length === 0) {
+    return '';
+  };
+  
+  // convert text to all lower case
+  let text = sample_text.toLowerCase();
+  let letter = text.slice(0, 1);
+
+  if ([letter]) {
```

Comment:
The square brackets are not necessary here and actually will make the if statement always return true. 

You'll also need to figure out if the particular character you're getting is in the counter object or not. Check out the object.hasOwnProperty method for a way to double check if a key exists in an object.